helpers/sqs_helper.py

```diff
@@ -0,0 +1,108 @@
+"""
+Helper module for sqs messages
+"""
+import json
+import os
+import sys
+import time
+import boto3
+from botocore.exceptions import ClientError
+
+# add project root to sys path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from helpers.base_helper import BaseHelper
+from conf import sqs_conf
+
+class SqsHelper(BaseHelper):
+    """
+    SQS Helper object
+    """
+
+    def get_sqs_client(self):
+        """
+        Return sqs_client object
+        :param self:
+        :return sqs_client: SQS client object
+        """
+        try:
+            sqs_client = boto3.client('sqs', config=sqs_conf.config)
```

Comment:
Wouldn't `sqs_client` just be self.client and wouldn't it get initialized in __init_-()? Is there a use case I am missing where you do not need the client object?